calculator.rb

```diff
@@ -0,0 +1,109 @@
+# calculator program that allows user to choose what operator, along with two numbers to calculate.
+
+# the program will run once while true and if the user wants to play again, they would need to type 'y' when prompted
+play_again = "y"
+
+
+# while will run true the first time and begin the program
+while play_again == "y"
+
+  puts """
+  Welcome to the calculator program. Which operator would you like to use?
+  1. add(+)
+  2. subtract(-)
+  3. multiply(*)
+  4. divide(/)
+
+  """
+
+# prompts the user to select which operator option they'd like to use.
+  print "Please choose one operator (name or symbol): "
+  user_input = gets.chomp
```

Comment:
I think there might be a more specific name for this variable than `user_input`, since there are several pieces of user input in this program. Something like `operation` might be a better choice.